modules/render.js

```diff
@@ -0,0 +1,14 @@
+import Books, { bookList } from './createbooks.js';
+
+export default class Render {
+  static renderBooks() {
+    const bookSection = document.querySelector('.box');
+    const bookListElement = document.createElement('div');
+    bookListElement.id = 'bookList';
+    bookList.forEach((book) => {
+      const bookElem = new Books(book.title, book.author, book.id);
+      const bookElement = bookElem.createBook();
+      bookSection.append(bookElement);
+    });
+  }
+}
```

Comment:
- [ ] Kindly fix the following  issue that makes your project not run as expected. I am not seeing anything once I click on the `List, Add new, Contact`

![image](https://user-images.githubusercontent.com/55635977/171210284-53ba33e0-f29f-4b8d-b060-6a6d7bf71aaa.png)

Reply:
Hello @Emmyn5600! Thanks for the review!

I believe there was an error with the Live version because the code works perfectly on my computer.
I've updated the [live server](https://mikemtzp.github.io/Awesome-books-es6/)

![image](https://user-images.githubusercontent.com/97407840/171214353-0cb71822-2a76-449f-b552-0a3e5d21e04a.png)
![image](https://user-images.githubusercontent.com/97407840/171214402-edfa445f-ed4f-4176-bff6-1fe24cd7a863.png)